test/EndToEnd/tests/PackTest.ps1

```diff
@@ -75,7 +75,7 @@ function Test-PackFromProjectWithDevelopmentDependencySet {
 
     # Act
     
-    & $context.NuGetExe pack $p.FullName -build -o $output
+    & $context.NuGetExe pack $p.FullName -build -OutputDirectory $output
```

Comment:
Why did we cut this option?
Isn't this a regression/breaking change?

Reply:
didn't cut this, there are two options which start with "o" now

Reply:
My question was, when did we cut it? 
Or was this option automagically resolved to the OutputDirectory one because there was only one option starting with o?

Reply:
this was automagically resolved to the OutputDirectory one because there was only one option starting with o

Reply:
Ah, ok.
Makes sense then.